### PR TITLE
Release ver.1.0

### PR DESCRIPTION
**Pull request by TwinGan**
image
1. markdown scripts
2. change some test cases input way
All tests passed
@FedericoTartarini

### CODE REVIEW

File: tests/conftest.py [hunk 1]
```diff
@@ -74,7 +74,7 @@ def is_equal(a, b, tolerance=1e-6):
         if a.dtype.kind in "UOS":  # U = unicode, O = objects, S = string
             return np.array_equal(a, b)
         else:
-            b = np.where(b is None, np.nan, b)  # Replace None with np.nan
+            b = np.where(b == None, np.nan, b)  # Replace None with np.nan
```
Comment (FedericoTartarini) [inline tests/conftest.py:77]: I think that we should keep b is None. PyCharm is suggesting me that it is better

File: tests/scripts_test_result.py [hunk 2]
```diff
@@ -0,0 +1,52 @@
+import subprocess
```
Comment (FedericoTartarini) [inline tests/scripts_test_result.py:1]: please explain how to run this file and how what the file does in a comment at the beginning of the file.

Comment (FedericoTartarini) [reply tests/scripts_test_result.py:1]: also please explain how to run it in the readme

File: tests/conftest.py [hunk 3]
```diff
@@ -128,4 +128,4 @@ def validate_result(result, expected_output, tolerance: dict):
             assert is_equal(_result, _expected_output, _tolerance)
         except Exception as e:
             print(f"Expected {_expected_output}, got {_result}\nError: {str(e)}")
-            raise
+            raise
```
Comment (FedericoTartarini) [inline tests/conftest.py:131]: This code looks the same for me.

File: tests/test_ankle_draft.py [hunk 4]
```diff
@@ -15,7 +15,6 @@ def test_ankle_draft(get_test_url, retrieve_data):
         inputs = entry["inputs"]
         outputs = entry["outputs"]
         result = ankle_draft(**inputs)
-
```
Comment (FedericoTartarini) [inline tests/test_ankle_draft.py:18]: no need to remove this line

File: tox.ini [hunk 5]
```diff
@@ -48,7 +48,7 @@ deps =
     numba
     pandas
 commands =
-    {posargs:pytest --cov --cov-report=term-missing -vv tests}
+    {posargs:pytest --cov --cov-report=term-missing -vv tests --ignore=./tests/scripts_test_result.py}
```
Comment (FedericoTartarini) [inline tox.ini:51]: please explain in this pull request what this code does

**Review by FedericoTartarini — CHANGES_REQUESTED** [review]
I left a few minor comments for you to address before I can accept the Pull request

File: tox.ini (hunk 5 above)
Comment (TwinGan) [reply tox.ini:51]: ignore scripts_test_result.py file when running tox

**Comment by TwinGan** [comment]
@FedericoTartarini, Professor, all the questions have been solved. Please review it again.

**Review by FedericoTartarini — APPROVED** [review]

**Pull request merged**